docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Data Storage and Controls
+sidebar_position: 60
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What data Cypress Cloud stores
+- What controls exist to manage data sent to and viewed in Cypress Cloud
+
+:::
+
+## Data Storage
+
+:::caution
+
+Please refer to the [Cypress Cloud Terms of Use](https://cloud.cypress.io/terms-of-use) and our [Security & Compliance](https://www.cypress.io/security/) guide for more specifics around reasonable use of Cypress Cloud, data storage and security at Cypress.
+
+:::
+
+When you run Cypress via `cypress run` and pass the `--record` flag, the following data is stored in Cypress Cloud for every run:
+
+- Standard output in the terminal
+- Test results
+- Test definitions
+- Cypress configuration (minus Cypress environment variables)
+- Screenshots
+- Videos
+- OS environment variables related to CI and git information
+
+When [Test Replay](/guides/cloud/debugging/test-replay) capturing is enabled, the additional data outlined below will be sent to Cypress Cloud. [See our guide on disabling Test Replay.](/guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay)
+
+- The rendered DOM and CSS styles for the application under test
+- Cypress commands and events represented in the Command Log
+- Network traffic within your application under test
+- Browser console logs
+
+Cypress Cloud does not capture anything related to the code of your application under test or the code within any repositories that are associated with your project.
+
+You, the customer, own and are responsible for your test content. We trust that customers will make decisions about the appropriateness of data to use in testing and will avoid use of PII, PHI, or other types of protected information. Cypress makes commercially reasonable efforts to protect confidentiality of information that is provided to Cypress Cloud.
+
+## Cypress Cloud data controls
+
+We understand that controlling what data is sent to Cypress Cloud is important to our users. Cypress provides a number of ways to control what data is sent to Cypress Cloud.
```

Comment:
```suggestion
We understand that controlling what data is sent to Cypress Cloud is important to our users. Cypress provides a number of ways to control this data transfer.
```

Attempting to avoid repetition.